src/EventTickets/resources/components/EventTicketsHeader.tsx

```diff
@@ -1,9 +1,9 @@
-import moment from 'moment';
+import {format} from 'date-fns';
 
 export default function EventTicketsHeader({title, startDateTime}) {
-    const fullDate = moment(startDateTime).format('dddd, MMMM Do, h:mma z');
-    const day = moment(startDateTime).format('DD');
-    const month = moment(startDateTime).format('MMM');
+    const fullDate = format(startDateTime, 'EEEE, MMMM do, hh:mmaaa');
+    const day = format(startDateTime, 'dd');
+    const month = format(startDateTime, 'EEE');
```

Comment:
Looking at the documentation, shouldn't the month format be `MMM`?

![image](https://github.com/impress-org/givewp/assets/10858303/4af74966-d64a-4115-aa87-108fe9aa4545)

![image](https://github.com/impress-org/givewp/assets/10858303/0f5d92a0-127d-41f9-9b0b-cb3d4e6a1c69)

Reply:
Resolved: 978a4412bcae775120c30caec79dffd6221ea87b